Assets/css/style.css

```diff
@@ -65,6 +63,114 @@ textarea {
   width: 100%;
 }
 
+/* CLASS BASE SECTION */
+
+.clear {
+  clear: both;
+}
+
+.contact-container {
+  align-items: center;
+  display: flex;
+  min-height: 400px;
+  flex-direction: row;
+}
+
+.container {
+  box-sizing: border-box;
+  margin: 0 auto;
+  max-width: 1080px;
+  padding: 10px;
+  width: 100%;
+}
+
+.features {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  column-gap: 30px;
+  margin: 50px 0 30px 0;
+  width: 100%;
+}
+
+.features .feature-item {
+  margin-bottom: 20px;
+}
+
+.features .icon-skill {
+  height: 20px;
+  margin-right: 10px;
+  position: relative;
+  vertical-align: baseline;
+  width: 20px;
+}
+
+.header-flex {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+.lang-tag {
+  list-style: none;
+}
+
+.latest-work {
+  display: grid;
+  grid-template-columns: auto auto;
+  column-gap: 30px;
+  margin-bottom: 30px;
+  width: 100%;
+}
+
+.latest-work .lang-tag {
+  list-style: none;
+  margin-bottom: 20px;
+}
+
+.latest-work-image {
+  height: auto;
+  position: relative;
+  width: 100%;
+}
+
+.main-presentation {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
```

Comment:
- [ ] Kindly adjust your styles to slightly move the left text to align with the Logon on the navbar in vertical view.

- From the look of your design below things are not aligned properly on desktop screens

![image](https://user-images.githubusercontent.com/50941074/165584727-fffe836b-b5b0-41e0-9ffe-dc113f6fefc1.png)

- See how things are aligned in the Figma design.
![image](https://user-images.githubusercontent.com/50941074/165584927-e11bc8a3-e372-40f1-a95f-b662e93c6619.png)

- [ ] Please fix that 👍